express-intro/server.ts

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+const app = express();
+
+app.listen(8080, () => {
```

Comment:
By convention, we usually put the `listen` last. It doesn't matter, but it makes reading the code more logical.